auth0/v3/authentication/base.py

```diff
@@ -16,8 +16,8 @@ def get(self, url, params={}, headers={}):
     def _process_response(self, response):
         text = json.loads(response.text) if response.text else {}
 
-        if 'error' in text:
-            raise Auth0Error(status_code=text['error'],
-                             error_code=text['error'],
-                             message=text['error_description'])
+        if response.status_code >= 400:
+            raise Auth0Error(status_code=text.get('error', ''),
```

Comment:
@ravoro This still misses that the status code reported in the Auth0Error should be that actual status code of the response.  Thus, this line should be: `raise Auth0Error(status_code=response.status_code,`

Reply:
@beck3905 Good point! I've just amended the PR with the fix.